core/cloudflow-akka-testkit/src/main/scala/cloudflow/akkastream/testkit/javadsl/OutletTap.scala

```diff
@@ -48,7 +50,7 @@ case class ProbeOutletTap[T](outlet: CodecOutlet[T])(implicit system: ActorSyste
       .alsoTo(
         Flow[PartitionedValue[T]]
           .map(pv ⇒ Pair(pv.key, pv.value))
-          .to(Sink.actorRef[Pair[String, T]](probe.getTestActor, Completed))
+          .to(Sink.actorRef[Pair[String, T]](probe.getTestActor, Completed, Failed))
```

Comment:
Completed -> Failed ?

Reply:
it's new API you have to provide what the failure message will be too.

Reply:
Ignore it, in the split lane it looked like you replaced Completed with Failed.